src/main/java/com/cmsc495/ticketsystem/service/TicketService.java

```diff
@@ -23,26 +23,52 @@ public class TicketService {
     @Autowired
     private TicketRepository ticketRepository;
 
-    public void changeTicketAndStatus(Long ticketId, String status, String noteContent) {
-        TicketModel ticket = ticketRepository.findById(ticketId).orElseThrow(()
-                -> new RuntimeException("Ticket not found"));
+    /**
+     * Changes the status of a ticket and adds troubleshooting notes.
+     * @param ticketId ID of the ticket to be updated.
+     * @param status New status for the ticket.
+     * @param noteContent Troubleshooting notes to be added.
+     * @return The updated TicketModel object.
+     */
+    public TicketModel changeTicketAndStatus(Long ticketId, String status, String noteContent) {
```

Comment:
Do you have a reasoning for changing the return type of the changeTicketAndStatus function? As far as I can tell the return type is never utilized.